sale_blanket_order_line_position/models/blanket_order_line.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import models
+
+
+class BlanketOrderLine(models.Model):
+    _name = "sale.blanket.order.line"
+    _inherit = ["sale.blanket.order.line", "order.line.position.mixin"]
+
+    def _add_next_position_on_new_line(self, vals_list):
```

Comment:
looks like this code is almost exactly the same than the one on `sale.order`.
Could it be factored?
You could use `self._name` and `self._table` perhaps.